### ISSUE TITLE
Imported CCDA Binaries

### ISSUE THREAD

**Issue by mikeseale** [virencehealth/apiserver.virencehealth.com#88]
What is the proper way to pull the binary data from a CCDA that has been imported into Centricity from an outside facility.  

**Example:**  
A clinic receives and imports a CCDA from a nearby hospital, and I would like to pull the binary content of that CCDA to display in an app.  For normal (RTF) documents within Centricity, I would use:
**GET [base]fhir/Binary/DOC-{SDID}** 

Where SDID is the document's ID found in the DocumentReference lookup.  However, when I do this for any documents that store their data as XML in the DOCDATA2 table, it will not return any results.

**Comment by Torin-Shepard-athena** [comment]
Use GET [base]/fhir/Binary/DD-(SDID) to get XML documents.

In theory, the answer to this question could have been discovered through DocumentReference query for a patient known to have an imported CCDA in their chart.
GET [base]/fhir/DocumentReference?patient=PID&_count=50
The Bundle of DocumentReference resources will contain references to Binary resources, and each will have the proper prefix depending upon the content.

The Binary resource derives from multiple object types.  We realize this information is missing from [API Resources](https://mydata.athenahealth.com/fhirapidoc) documentation and plan to add it in the future.

| ID Prefix | Information in Resource |
| --- | --- |
| Photo	| CPS PatientProfile Picture, EMR PERSON_PICT |
| DOC | Chart documents created by CPS/CEMR |
| DD | CCDAs imported into CPS/CEMR |
| PPA | CPS PatientProfileAttachment |
| PVA | CPS PatientVisitAttachment |

**Comment by mikeseale** [comment]
Thanks for the reply, Torin.  We actually tried using DD, but we get the following error:

"diagnostics": "Resource Binary/DD-1890037170911360 is not known"

However, you are correct that it will show the correct URL within DocumentReference, and we just needed to parse that out instead of trying to call this directly.  It turns out that the correct ID to use for DD- is the primary key of the table DOCDATA2 instead of the SDID.

GET [base]/fhir/Binary/DD-{**DDID**}
